src/gates_function.jl

```diff
@@ -118,6 +155,20 @@ function create_gates(s, ψ, N, B, N_sites, Δx, Δm², p, x, Δp, theta_nu, sha
             # println((Δm²))/(2 *hbar * p_mod[1])
             # @assert imag(hj) == ((Δm²))/(2 *hbar * p_mod[1])
             # println("Gj= ",Gj)            # has_fermion_string(hj) = true
+            if theta_nu == 0 ||  theta_nu == π/4 || theta_nu == π/2 
```

Comment:
These if statments detect which test is being run in an esoteric way. I think they should be removed.